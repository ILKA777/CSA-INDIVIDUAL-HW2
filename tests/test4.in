10
fds11234ds
