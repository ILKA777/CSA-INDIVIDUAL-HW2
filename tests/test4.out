Введите первоначальный размер строки:
Введите строку:
Количество : 1 - 2
Количество : 2 - 1
Количество : 3 - 1
Количество : 4 - 1
Других цифр в строке нет

